notebooks/01_webscraping_py.ipynb

```diff
@@ -0,0 +1,76 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Web scarpping code"
+   ]
```

Comment:
I thought we decided not to use `Jupyter notebooks` - kindly switch to `Rmd` files.

Reply:
I have started this earlier before using Rstudio , copy and pasting these code to that will be a time consuming task so I continued to jupyter and moreover some library like folium and others were not giving proper output in the Rstudio so decided to shift jupyter . They were giving the location of the output but was not showing the graph , so have to parse the output location to HTML and then print it , so again it was possible but tedious.  For other projects I will prefer Rstudio but consider this one an exception .👍

Reply:
As of now no time to this project from jupyter to R that why not added this one in the Data Science organization in the first place ..

Reply:
Ok bro :)